android/.project

```diff
@@ -1,34 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<projectDescription>
-	<name>nearby_connections</name>
-	<comment>Project android created by Buildship.</comment>
-	<projects>
-	</projects>
-	<buildSpec>
-		<buildCommand>
-			<name>org.eclipse.jdt.core.javabuilder</name>
```

Comment:
Why are files in .settings, .project, . classpath deleted?

I think they are supposed to be kept in version control?

Reply:
I created a new flutter plugin project with the latest settings, and then ported over your java code. These seems to not exist in the the new project.

Reply:
cool